components/BottomNavigation/tests/unit/BottomNavigationTests.m

```diff
@@ -0,0 +1,85 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialBottomNavigation.h"
+#import "../../src/private/MDCBottomNavigationItemView.h"
+
+@interface MDCBottomNavigationBar (Testing)
+@property(nonatomic, strong) NSMutableArray<MDCBottomNavigationItemView *> *itemViews;
+@end
+
+@interface BottomNavigationTests : XCTestCase
+@property(nonatomic, strong) MDCBottomNavigationBar *bottomNavBar;
+@end
+
+@implementation BottomNavigationTests
+
+- (void)setUp {
+  self.bottomNavBar = [[MDCBottomNavigationBar alloc] init];
+}
+
+- (void)tearDown {
+  self.bottomNavBar = nil;
+}
+
+- (void)testInit {
+  // When
+  MDCBottomNavigationBar *bar = [[MDCBottomNavigationBar alloc] init];
```

Comment:
maybe worth testing self.bottomNavBar for XCAssertNotNil, which is allocated in setUp, rather than creating another MDCBottomNavigationBar? Not sure it really makes a difference, but bringing it up for conciseness sake.

Reply:
Agreed it's not the least amount of code to achieve the result. I felt it was OK to leave it separate in case we change to a different initializer in the future.